hw-7/7.1/style.css

```diff
@@ -0,0 +1,120 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Open Sans', sans-serif;
+}
+
+.wrapper {
+    width: 950px;
+    position: absolute;           
+    top: 50%;  
+    left: 50%;                       
+    transform: translate(-50%, -50%);
+    height: 100%;
+}
+
+.header {
+    text-transform: uppercase;
+    letter-spacing: 5px;
+    text-align: center;
+    line-height: 30px;
+    margin-bottom: 100px;
+}
+
+.header + p {
+    font-size: 12px;
+    color: #898f99;
+}
+
+.header h1 {
+    font-size: 48px;
+    color: #0ec8cb;
+    margin-bottom: 35px;
+    margin-top: 30px;
+    
+}
+
+.head-text {
+    font-size: 18px;
+    letter-spacing: 0;
+    color: #898f99;
+    width: 60%;
+    margin: 0 auto;
+}
+
+/* не працює...
+section:nth-child(1) {
+    
+}
+
+section:last-of-type {
+    font-size: 18px;
+    letter-spacing: 0;
+}
+*/
+.blocks {
+    font-size: 0;
+    
+}
+
+.block-item {
+    /*height: 420px;*/
+    width: 230px;
+    display: inline-block;
+    vertical-align: top;
+    text-align: center;
+    padding: 60px 35px 0 35px;
+    background-color: #f7f7f7;
+    border: 1px solid #ededed;
+}
+.img-wrapper {
+    height: 65px;
+    margin-bottom: 50px;
+}
+
+.hover-img {
+    display: none;
+}
+
+.h2-wrapper {
+    height: 45px;
+    margin-bottom: 30px;
+}
+
+.block-item h2 {
+    font-size: 21px;
+    color: #1b1b1b;
+    line-height: 21px;
+}
+
+.block-item p {
+    font-size: 14px;
+    line-height: 22px;
+    color: #777777;
+    padding-bottom: 90px;
+    /*text-align: justify; ???*/
+}
+
+.block-item:hover {
+    background-color: #0ec8cb;
+    color: #fff;
+    border: 1px solid grey;
+}
+
+.block-item:hover .v-img {
+    display: none;
+}
+
+.block-item:hover .hover-img {
+    display: inline-block;
+}
+
+.block-item:hover h2 {
+    color: #fff;
+}
+
+.block-item:hover p{
```

Comment:
we can merge these item intone one  .block-item:hover h2,  .block-item:hover p{color: #fff;}